### ISSUE TITLE
Let Design Jam be a general event

### ISSUE THREAD

**Issue by anubhavp28** [IIITKalyaniFOSC/design-jam#3]
Great work on README.md @Satyam-TG.

> Design Jam 2018 is basically a logo designing competition for IIIT Kalyani FOSC and is being organized by the same. IIIT Kalyani FOSC is the Free and Open Source Community of the college. It consists of people who love open source and is currently in the beginning stage. And, as a matter of fact, it needs a beautiful logo. <

Well, I would say, make Design Jam - FOSC's annual design competition. Change the title of README.md to just "Design Jam". This year we kept the design jam to be about FOSC logo (since we need one). Can you make the whole "What is this?" section not about logo designing? and remove "What is this?" as header, keep that paragraph directly below the main heading.

Add another paragraph about logo designing.

@zeus0789 @PulkitMishra What do you guys say?

**Comment by satyamtg** [comment]
@anubhavp28 , good idea. A better idea would be to make a different branch each year and update the README.md as you said. We can also add the winning entry each year to the repository. What do you say?

**Comment by anubhavp28** [comment]
@Satyam-TG I like your proposed idea of using a different branch each year. :+1:

**Comment by abhishekti7** [comment]
@Satyam-TG Excellent idea man. Adding the winning entry of each year to the repository sounds like the thing to do. Great work!

**Comment by PulkitMishra** [comment]
Agree with @Satyam-TG on this.

**Comment by satyamtg** [comment]
How about creating a master branch for all winning entries and general instructions (a general README about what is Design Jam, and maybe a future website)  and letting the 2018 branch contain details about this year ?

**Comment by Pratik57** [comment]
It means every year FOSC logo change as per winner designed logo ??
@Satyam-TG

**Comment by satyamtg** [comment]
Nope. Every year design jam will be held for designing a different thing. It's only this year that we are designing the FOSC logo. It may be something else next year.
